packages/java/endpoint/src/main/java/com/vaadin/hilla/route/ClientRouteRegistry.java

```diff
@@ -144,18 +145,18 @@ private String removeTrailingSlash(String path) {
     public synchronized void registerClientRoutes(
             DeploymentConfiguration deploymentConfiguration,
             LocalDateTime lastUpdated) {
-        var viewsJsonAsResource = getViewsJsonAsResource(
+        var fileRoutesJsonAsResource = getFileRoutesJsonAsResource(
                 deploymentConfiguration);
-        if (viewsJsonAsResource == null) {
+        if (fileRoutesJsonAsResource == null) {
             LOGGER.debug(
-                    "No {} found under {} directory. Skipping client route registration.",
+                    "No {} found under {} directory yet. Skipping client route registration.",
```

Comment:
It is weird to put "yet" in the error message when you are not sure it will ever be found.

Reply:
```suggestion
                    "No {} found under {} directory. Skipping client route registration.",
```